patch-desktop-filename.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import tempfile
+import asarPy
+import shutil
+import json
+import sys
+import os
+
+
+def main() -> None:
+    asar_path = sys.argv[1]
+    extract_dir = tempfile.mktemp()
```

Comment:
`with tempfile.TemporaryDirectory(...)`, then you can remove `shutil` usage.

Reply:
This does not work here. `extract_asar` wants a directory that not exists yet as argument.